rrfs-test/IODA/python/bufr2ioda_msonet_prepbufr.py

```diff
@@ -0,0 +1,562 @@
+
+    # Observation Type: windEastward
+    obsspace.create_var('ObsType/windEastward', dtype=obstyp.dtype, fillval=obstyp.fill_value) \
+        .write_attr('long_name', 'Observation Type') \
+        .write_data(typ_uob)
+
+    # Observation Type: windNorthward
+    obsspace.create_var('ObsType/windNorthward', dtype=obstyp.dtype, fillval=obstyp.fill_value) \
+        .write_attr('long_name', 'Observation Type') \
+        .write_data(typ_vob)
+
+    # MetaData: Latitude
+    obsspace.create_var('MetaData/latitude', dtype=lat.dtype, fillval=lat.fill_value) \
+        .write_attr('units', 'degrees_north') \
+        .write_attr('valid_range', np.array([-90, 90], dtype=np.float32)) \
+        .write_attr('long_name', 'Latitude') \
+        .write_data(lat)
+
+    # MetaData: Longitude
+    obsspace.create_var('MetaData/longitude', dtype=lon.dtype, fillval=lon.fill_value) \
+        .write_attr('units', 'degrees_east') \
+        .write_attr('valid_range', np.array([-180, 180], dtype=np.float32)) \
+        .write_attr('long_name', 'Longitude') \
+        .write_data(lon)
+
+    # MetaData: Data Provider Origin
+    obsspace.create_var('MetaData/dataProviderOrigin', dtype=prvstg.dtype, fillval=prvstg.fill_value) \
+        .write_attr('long_name', 'Mesonet Provider ID String') \
+        .write_data(prvstg)
+
+    # MetaData: Data Provider Sub Origin
+    obsspace.create_var('MetaData/dataProviderSubOrigin', dtype=sprvstg.dtype, fillval=sprvstg.fill_value) \
+        .write_attr('long_name', 'Mesonet SubProvider ID String') \
+        .write_data(sprvstg)
+
+    # MetaData: Station Identification
+    obsspace.create_var('MetaData/stationIdentification', dtype=sid.dtype, fillval=sid.fill_value) \
+        .write_attr('long_name', 'Station Identification') \
+        .write_data(sid)
+
+    # MetaData: Station Elevation
+    obsspace.create_var('MetaData/stationElevation', dtype=elv.dtype, fillval=elv.fill_value) \
+        .write_attr('units', 'm') \
+        .write_attr('long_name', 'Station Elevation') \
+        .write_data(elv)
+
+    # MetaData: Pressure
+    obsspace.create_var('MetaData/pressure', dtype=pob.dtype, fillval=pob.fill_value) \
+        .write_attr('units', 'Pa') \
+        .write_attr('long_name', 'Pressure') \
+        .write_data(pob)
+
+    # MetaData: Height of Observation
+    obsspace.create_var('MetaData/height', dtype=zob.dtype, fillval=zob.fill_value) \
+        .write_attr('units', 'm') \
+        .write_attr('long_name', 'Height of Observation') \
+        .write_data(zob)
+
+    # MetaData: Datetime
+    obsspace.create_var('MetaData/dateTime', dtype=dateTime.dtype, fillval=dateTime.fill_value) \
+        .write_attr('units', 'seconds since 1970-01-01T00:00:00Z') \
+        .write_attr('long_name', 'Datetime') \
+        .write_data(dateTime)
+
+    # MetaData: timeOffset
+    obsspace.create_var('MetaData/timeOffset', dtype=dhr.dtype, fillval=dhr.fill_value) \
+        .write_attr('units', 's') \
```

Comment:
Just putting a note here for when you comment out the hour conversion to update this too.

Reply:
@delippi as explained above, no need to change.